junit5/src/main/java/com/linecorp/armeria/testing/junit5/server/ServerExtension.java

```diff
@@ -300,6 +314,21 @@ public final ServiceRequestContextCaptor requestContextCaptor() {
         return contextCaptor;
     }
 
+    /**
+     * Returns the {@link WebClient}.
+     */
+    public WebClient webClient() {
+        return delegate.webClient();
+    }
+
+    /**
+     * Creates a {@link WebClient} each time with {@link WebClientBuilder}.
+     */
+    public WebClient webClient(WebClientBuilder webClientBuilder) {
```

Comment:
I think this really doesn't do anything because the `delegate` just calls `webClientBuilder.build()` in it.
Could you tell me when we need this?

Reply:
I added it because of [this comment](https://github.com/line/armeria/pull/3890#discussion_r734364586). Did I do something different than the intent of this comment?

Reply:
I'm not sure what's the intention but like I said the only thing this method does is just `webClientBuilder.build()`. 😅 
Users just can build the client instead of calling this method.
@ikhoon Could you clarify your intention, please? 😄

Reply:
Oops... my previous comment was incorrect.
I thought:
```java
public WebClient webClient(Consumer<WebClientBuilder> webClientBuilder) { ... }

// Create a custom WebClient with the default URI and TLS configure.
var webClient = server.webClient(builder -> {
   builder.decorator(LoggingClient.newDecorator())
          .responseTimeoutMillis(0);
});
```

Reply:
If it looks overkill, `ServerExtension.configureWebClient(WebClientBuilder)` seems enough. 😄

Reply:
Ah ha, that looks useful. 😄

Reply:
I've edited it a bit 😄